src/store/Network.ts

```diff
@@ -225,6 +231,18 @@ export default {
       if (oldGenerationHash != networkModel.generationHash) {
         dispatch('account/NETWORK_CHANGED', {}, { root: true })
         dispatch('statistics/LOAD', {}, { root: true })
+        console.log(oldGenerationHash, networkModel.generationHash, currentProfile.generationHash)
```

Comment:
If this log is necessary, can it be formatted in a more descriptive way? Or even better, we can think about using diagnostic/ADD_DEBUG or diagnostic/ADD_LOG actions If we want user to be able to see in the console/logs(wallet desktop->top menu->console).

Reply:
It's not necessary, that was for testing purposes.